gbpservice/nfp/orchestrator/drivers/vyos_orchestration_driver.py

```diff
@@ -128,20 +127,22 @@ def get_network_function_device_config_info(self, device_data,
 
         return {
             'info': {
-                'version': 1
+                'service_vendor': (device_data['service_details'
+                                               ]['service_vendor'].lower()),
+                'service_type': (device_data['service_details'
+                                             ]['service_type'].lower())
             },
             'config': [
                 {
                     'resource': 'interfaces',
-                    'kwargs': {
-                        'vm_mgmt_ip': device_data['mgmt_ip_address'],
-                        'service_vendor': device_data['service_vendor'],
+                    'resource_data': {
+                        'mgmt_ip': device_data['mgmt_ip_address'],
                         'provider_ip': provider_ip,
                         'provider_cidr': provider_cidr,
-                        'provider_interface_position': 2,
+                        'provider_interface_index': 2,
                         'stitching_ip': consumer_ip,
                         'stitching_cidr': consumer_cidr,
-                        'stitching_interface_position': 3,
+                        'stitching_interface_index': 3,
```

Comment:
Let us remove the rule_info dict from the API input. It is unused in configurator.